docs/lib/app.js

```diff
@@ -65,6 +65,7 @@ export default (locals, callback) => {
           <link rel=icon href=/assets/favicon.ico>
           <link rel="stylesheet" href="/assets/style.css"/>
           <link rel="stylesheet" href="/assets/docs.css"/>
+          <link rel="stylesheet", href="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.css" />
```

Comment:
Please update end point to use CDNJS 

https://cdnjs.com/libraries/docsearch.js

Reply:
The one used is what Algolia (the search provider) provided. It has a feature which cdnjs does not, it can use semver to provide the latest version.
It has some pluses and minuses, such as since the version and this the content can change, we cannot use SRI.

Reply:
Both use Cloudflare as a backend and I would argue that CDNJS has best global performance and features (aside from ~latest semver) which has been deprecated by most major CDNs (google, CDNJS, etc) because it's bad practice and I realize developer just pointed link to JSDELIVR ... I'd still say it's okay using CDNJS.

Either way more of a recommendation to be consistent with our CDN offerings for main reactstrap.

Reply:
It's about cache hits and misses. Using what they recommend gives the best chance of a cache hit.
In the end, it really doesn't matter. I just want to get this feature added to the docs.